### PR TITLE
Redis pub/sub affinity system

### PR DESCRIPTION
**Pull request by RolandBc**
Hello Josh,
First pull request for redis affinity system. 
I started unit tests with jest but there are not ready yet, i'll push them in the next PR if it's ok for you.
Your comments on my code and my approach to integrate redis are welcome if you have time.
Thanks
Roland

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +85,60 @@ We throw in the constructor if we cannot contact the affinity server within `aff
...
+const zbcRedis = new RedisAffinity(ZEEBE_GATEWAY, { host: REDIS_HOST, password: REDIS_AUTH });
+
+zbcRedis.createWorkflowInstanceWithAffinity({
+        bpmnProcessId: workflowName,
+        variables: {
+        correlationKey,
+        userInputText: userInput,
+        chatFinished,
+        },
+        cb: (message) => res.send(message),
```
Comment (jwulf) [inline README.md]: This example demonstrates replying to a REST req with the outcome of the workflow? 

If so, we should probably make that explicit and wrap it in a REST route handler.

**Review by jwulf — CHANGES_REQUESTED** [review]
Just one small change in the README, I think.

File: README.md (hunk 1 above)
Comment (RolandBc) [reply README.md]: Yes your right! Modif done

**Review by jwulf — APPROVED** [review]
Awesome, thanks!

**Pull request merged**